gwpy/timeseries/io/gwf/framecpp.py

```diff
@@ -401,6 +401,7 @@ def read_frvect(vect, epoch, start, end, name=None, series_class=TimeSeries):
     dim = vect.GetDim(0)
     dx = dim.dx
     x0 = dim.startX
+    xunit = dim.GetUnitX()
```

Comment:
Is there any risk that the units for "x" do not exist? I ask because of the line `unit = vect.GetUnitY() or None`

Reply:
According to the GWF spec there is no risk that the unit is not specified at all (the variable doesn't exist), but there is a possibility that it is set to the empty string, which is probably accepted by the constructor, but is actually an invalid value. I have updated this patch to include the `or None` catch just in case. Thanks.